ts/components/ButtonDefaultOpacity.tsx

```diff
@@ -1,14 +1,42 @@
 import { Button, NativeBase } from "native-base";
 import * as React from "react";
+import customVariables from "../theme/variables";
 
 const defaultActiveOpacity = 1.0;
+
+type Props = NativeBase.Button & React.Props<Button>;
+
+/**
+ *
+ * @param height
+ */
+const calculateSlop = (height: number): number => {
```

Comment:
Since this function may be reused on other components what do you think of isolating it in a util such as `utils/ui.ts`?

Reply:
My idea was to remove all the code inside this file as soon as will be a refactoring of the basic components. 
Perhaps it would be better to leave it there to facilitate its removal and if it should serve elsewhere, move it to a utility file. What you think?

Reply:
> My idea was to remove all the code inside this file as soon as will be a refactoring of the basic components.
> Perhaps it would be better to leave it there to facilitate its removal and if it should serve elsewhere, move it to a utility file. What you think?

I'm agree.
Until we don't refactor these partes it's convenient to add code in a focused otherwise it will be more difficult to remove/refactor if it is spread around the project